org.kde.lokalize.json

```diff
@@ -0,0 +1,72 @@
+{
+    "id": "org.kde.lokalize",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "lokalize",
+    "rename-icon": "lokalize",
+    "finish-args": 
+    [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kross",
+            "buildsystem": "cmake-ninja",
+            "config-opts": 
+            [ 
+                "-DENABLE_TESTING=OFF"
+            ],
+            "sources": 
+            [ 
+                { 
+                    "type": "git",
```

Comment:
any particular reason for using the git repo here instead of the tarball?

Reply:
I did not find it on https://invent.kde.org/frameworks/kross/-/releases. Can I use from https://github.com/KDE/kross/releases ?

Reply:
Don't use github, it makes no sense, it's just a mirror. Use https://download.kde.org/stable/frameworks/

Reply:
Can not found Kross under https://download.kde.org/stable/frameworks/

Reply:
Sorry forgot to mention it's inside portingAids, i.e.

https://download.kde.org/stable/frameworks/5.69/portingAids/kross-5.69.0.tar.xz

Reply:
https://github.com/flathub/flathub/pull/1603/commits/9ca1a3efba615ca5e3bf82323c3b5f8d2de30e17